package.json

```diff
@@ -37,13 +37,17 @@
     "react": "^0.14.0"
   },
   "devDependencies": {
-    "babel": "^5.8.21",
+    "babel-cli": "^6.1.18",
+    "babel-core": "^6.1.21",
     "babel-eslint": "^4.1.1",
-    "babel-plugin-react-intl": "^1.0.0-beta-5",
-    "babelify": "^6.1.3",
-    "browserify": "^11.0.1",
-    "browserify-shim": "^3.8.10",
-    "envify": "^3.4.0",
+    "babel-plugin-react-intl": "^2.0.0",
+    "babel-plugin-transform-object-rest-spread": "^6.1.18",
+    "babel-preset-es2015": "^6.1.18",
+    "babel-preset-es2015-rollup": "^1.0.0",
+    "babel-preset-react": "^6.1.18",
+    "babelify": "^7.2.0",
+    "browserify": "^12.0.1",
```

Comment:
Can't both of these + the shim get removed now that Rollup is rolled in? :point_left: see what I did there :)